src/Pages/Templates/TemplateDetails/components/UseTemplate/CurlTab.tsx

```diff
@@ -79,7 +79,7 @@ const CurlTab = ({ tabContentRef }: Props) => {
               clickTip='Copied'
               variant={ClipboardCopyVariant.expansion}
             >
-              {'curl -o /etc/yum.repos.d/template.repo  https://cert.console.redhat.com/api/content-sources/v1/templates/' +
+              {'curl --cert /etc/pki/consumer/cert.pem  --key /etc/pki/consumer/key.pem -o /etc/yum.repos.d/template.repo  https://cert.console.redhat.com/api/content-sources/v1/templates/' +
```

Comment:
Do we have tests for this - for the whole "use templates"?

Reply:
i don't know of any iqe/playwright tests.  They really have to be tested on a client, i'll file a ticket to at least test that downloading a repo file and consuming content that way works, as well as using the api to assign a template to a system.